Assets/Scripts/Interactable.cs

```diff
@@ -58,4 +76,29 @@ void setIrritationScore(IrritationReaction irritation)
       Debug.Log($"{npc.name} didn't see you messing with {this}...");
     }
   }
+
+  protected virtual void MouthInteraction()
+  {
+    Debug.Log("MouthInteraction");
+  }
+
+  protected virtual void PawInteraction()
+  {
+    Debug.Log("PawInteraction");
+  }
+
+  protected virtual void BodyInteraction()
+  {
+    Debug.Log("BodyInteraction");
+  }
+
+  public void ShowTooltip()
+  {
+    interactableUI.ShowInteractionTip(true);
+  }
+
+  public void HideTooltip()
+  {
+    interactableUI.ShowInteractionTip(true);
```

Comment:
Swap this `true` to a `false`? I think it will make these tips go away after the interaction is complete.

![image](https://user-images.githubusercontent.com/21282705/133501823-71564318-3459-4310-90f5-ac52932f1484.png)